csp/impl/adaptermanager.py

```diff
@@ -24,6 +25,9 @@ def process_next_sim_timeslice(self, now):
         """
         return None
 
+    def engine_shutdown(self, msg):
```

Comment:
I see, the other method is hidden to add this formatting step
im hoping we can drop the need to pass formatted tracebacks and can just set PythonPassthrough directly

Reply:
Done, removed formatting method